CONTRIBUTING.md

```diff
@@ -5,7 +5,24 @@ pull requests. This document outlines some of the conventions on development
 workflow, commit message formatting, contact points and other resources to make
 it easier to get your contribution accepted.
 
-We gratefully welcome improvements to documentation as well as to code.
+We gratefully welcome improvements to issues and documentation as well as to code.
+
+## Working on issues
+
+If you like Flux and want to get involved in the team, a great way to get started
```

Comment:
Some people may take "team" to imply something more formal than what's meant here. To avoid introducing this ambiguity, you could omit the mention here and in the next para. (It would be good to retain the welcoming message at the end, if it can be done)